test/container/CMakeLists.txt

```
@@ -6,6 +6,12 @@
 #                  copied verbatim in the file "LICENSE"                       #
 ################################################################################
 
+include(GNUInstallDirs)
+
+if(NOT DEFINED FairSoft_CONTAINER_DIR)
+  set(FairSoft_CONTAINER_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}")
```

1. In FairMQ, ODC, etc I named this kind of "project-specific install tree path-component variables" `PROJECT_<component>[_]DIR`. In the very early times I had the concrete project name in there, but then realized it is always the same pattern and in one case was even annoying because I changed the project name capitalization and had to change now all those variables to keep it consistent. If it did not cross your mind before, maybe you want to consider, if `PROJECT_CONTAINER_DIR` would be suitable here, too. But I see a small but significant? difference to the other repos: In this case you allow the user to customize it, in the other repos the user can only customize the `CMAKE_INSTALL_<component>[_]DIR` part.

2.
```suggestion
  set(FairSoft_CONTAINER_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME_LOWER}")
```
If we do not have `PROJECT_NAME_LOWER` yet, please define it in the main CMakeLists.txt somewhere near and after the `project()` command (`STRING(TO_LOWER ...` or similar was the command).

---

Both should be addressed now.